package.json

```diff
@@ -72,7 +72,7 @@
   },
   "lint-staged": {
     "**/*.ts?(x)": [
-      "npm run lint:fix"
```

Comment:
the `lint-staged` section defines which scripts to run for each staged files.

In this case, if you had `file-a.tsx`, `file-b.tsx` and `file-c.html` staged, `lint-staged` would run:
`npm run lint:fix file-a.tsx file-b.tsx`

However, the `lint:fix` script executes the following `eslint --fix .`, meaning that everything in the repo was getting fixed every time we ran `git commit`.

In the case of everything getting run with prettier, it already worked properly.